src/fft.rg

```diff
@@ -144,133 +152,435 @@ function fft.generate_fft_interface(itype, dtype)
     return iface.get_tunable(DEFAULT_TUNABLE_NODE_COUNT)
   end
 
+  -- Get the number of GPUs
   __demand(__inline)
   task iface.get_num_local_gpus()
     return iface.get_tunable(DEFAULT_TUNABLE_LOCAL_GPUS)
   end
 
-  __demand(__inline)
+  --Task to return pointer to plan. Takes plan region and returns pointer to plan. Calls get_base_plan made above.
+   __demand(__inline)
```

Comment:
I think this is not part of the public interface? To me, this comment seems to recapitulate the task signature, which makes it non-helpful. If it were a public interface, we'd want to document it (ideally with a description that is more helpful), but I think we just don't need to do that at all in this case.

Reply:
Removed this, along with other comments for non-public tasks